store/src/main/java/com/dropbox/android/external/store4/StoreBuilder.kt

```diff
@@ -87,14 +88,15 @@ private class RealStoreBuilder<Key : Any, Input : Any, Output : Any>(
     private val sourceOfTruth: SourceOfTruth<Key, Input, Output>? = null
 ) : StoreBuilder<Key, Output> {
     private var scope: CoroutineScope? = null
-    private var cachePolicy: MemoryPolicy? = StoreDefaults.memoryPolicy
+    private var cachePolicy: MemoryPolicy<Key, Output>? = StoreDefaults.memoryPolicy
 
     override fun scope(scope: CoroutineScope): RealStoreBuilder<Key, Input, Output> {
         this.scope = scope
         return this
     }
 
-    override fun cachePolicy(memoryPolicy: MemoryPolicy?): RealStoreBuilder<Key, Input, Output> {
+    override fun cachePolicy(memoryPolicy: MemoryPolicy<Key, Output>?):
+        RealStoreBuilder<Key, Input, Output> {
```

Comment:
nit, if this doesnt' fit the previous line i think it is better to use multi line method description.
```suggestion
    override fun cachePolicy(
        memoryPolicy: MemoryPolicy<Key, Output>?
    ) : RealStoreBuilder<Key, Input, Output> {
```